pkg/model/estimator/local/lr_test.go

```diff
@@ -243,7 +244,7 @@ var _ = Describe("Test LR Weight Unit", func() {
 			// Estimate Container Components Power using Linear Regression
 			modelWeightFilepath := config.GetDefaultPowerModelURL(types.DynPower.String(), types.ComponentEnergySource)
 			initModelURL := "https://raw.githubusercontent.com/sustainable-computing-io/kepler-model-db/main/models/v0.6/nx12/std_v0.6/rapl/DynPower/BPFOnly/SGDRegressorTrainer_1.json"
```

Comment:
Probably need to update the model here, this model is SGD, not sure it it if compatible with XGBoost.
Did you try to locally?

Reply:
Might be also good to change the name of the files and other references from `lr` to `xgb`.